### ISSUE TITLE
Search in path ending in space breaks custom editor %file param argument

### ISSUE THREAD

**Issue by lopezpa79** [dnGrep/dnGrep#242]
If the "Search in" path ends with a space (as sometimes happens when you copy and paste from a document), the space is included in the path when attempting to open with a custom editor and using the %file argument.

E.g. Search in "C:\My Folder ", will list results as 

```
Subfolder1\SomeFile001.txt
Subfolder2\SomeFile003.txt
```

And when you attempt to open the file with the custom editor, the file paths used become:

```
"C:\My Folder \Subfolder1\SomeFile001.txt" instead of "C:\My Folder\Subfolder1\SomeFile001.txt"
and 
"C:\My Folder \Subfolder2\SomeFile002.txt" instead of "C:\My Folder\Subfolder2\SomeFile002.txt"
```

**Comment by doug24** [comment]
This should be fixed in the 2.9.21 beta

**Comment by JVimes** [comment]
Verified fixed.
